playground/.storybook/main.ts

```diff
@@ -15,9 +15,6 @@ const config: StorybookConfig = {
     name: '@storybook-vue/nuxt',
     options: {},
   },
-  docs: {
```

Comment:
Shouldn't this go in the [playground/.storybook/main.ts](https://github.com/nuxt-modules/storybook/blob/64e9873e146cc94267f1215a8d625a6b023059cb/playground/.storybook/preview.ts) now according to https://storybook.js.org/docs/writing-docs/autodocs#set-up-automated-documentation?